### PR TITLE
Contrast Error in Issue #439

### PR DESCRIPTION
**Pull request by Chimatara**
I fixed the contrast error in issue #439 after  changing the color and background color, I ran a wave accessibility test and it came out with zero contrast error however, I couldn't save the changes to the _site folder as gitpod doesn't track it.

### CODE REVIEW

File: _data/ols-4-schedule.yaml [hunk 1]
```diff
@@ -221,7 +221,6 @@ weeks:
         - Create a GitHub repository
```
Comment (bebatut) [inline _data/ols-4-schedule.yaml:221]: Can you remove the changes in this file?

File: _sass/bulma/css/bulma.css [hunk 2]
```diff
@@ -388,8 +388,8 @@ a:hover {
 }
```
Comment (bebatut) [inline _sass/bulma/css/bulma.css:388]: Could you make the changes not in the `_sass/bulma/css/bulma.css` but in `css/custom.scss`?

File: _data/ols-4-schedule.yaml (hunk 1 above)
Comment (Chimatara) [reply _data/ols-4-schedule.yaml:221]: I have reverted the changes I made earlier.

https://github.com/Chimatara/open-life-science.github.io/blob/Revert-Changes/_data/ols-4-schedule.yaml

File: _sass/bulma/css/bulma.css (hunk 2 above)
Comment (Chimatara) [reply _sass/bulma/css/bulma.css:388]: Hi @bebatut I have tried making the changes using the css/custom.scss file but I am unable to find the class needed in the custom.scss file, I'm only able to find it in the _site css folder and the _sass folder. Is their another way I can do this?

Comment (bebatut) [reply _sass/bulma/css/bulma.css:388]: Everything you put the custom.css file will overwrite the css from Bulma folder. You can then define the class and add the components to overwrite there

**Comment by bebatut** [comment]
This was fixed in #500. Thanks

**Pull request closed without merging**